src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -61285,6 +61285,31 @@ static void G(object? x)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "z").WithLocation(9, 45));
         }
 
+        [Fact, WorkItem(55483, "https://github.com/dotnet/roslyn/issues/55483")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/55483 [](http://example.com/codeflow?start=32&length=45)

Is this test related to the issue? (It looks like we produce the same errors before this change?) If it's not related, consider removing the `[WorkItem]` attribute.

Reply:
IIRC, an intermediate version of this PR would crash only in this test.